src/locale/locales/pt-BR/messages.po

```diff
@@ -2631,7 +2631,7 @@ msgstr "Feed por {0}"
 
 #: src/components/StarterPack/Wizard/WizardListCard.tsx:55
 msgid "Feed toggle"
-msgstr ""
+msgstr "Alterar Feed"
```

Comment:
I think "Alternar" is the correct one here

Reply:
Ok